tsconfig-strict.json

```diff
@@ -98,8 +98,10 @@
     "extensions/interactions/AlgebraicExpressionInput/directives/algebraic-expression-input-rules.service.spec.ts",
     "extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-rules.service.ts",
     "extensions/interactions/DragAndDropSortInput/directives/drag-and-drop-sort-input-rules.service.spec.ts",
+    "extensions/interactions/LogicProof/directives/logic-proof-rules.service.ts",
+    "extensions/interactions/LogicProof/directives/logic-proof-rules.service.spec.ts",
     "core/templates/domain/classifier/classifier.model.ts",
-    "core/templates/domain/classifier/classifier.model.spec.ts"
+    "core/templates/domain/classifier/classifier.model.spec.ts",
```

Comment:
Oh, I think this unsorted thing got in a previous PR. @parulpriyedarshani can please move the 
```
"core/templates/domain/classifier/classifier.model.ts",
    "core/templates/domain/classifier/classifier.model.spec.ts"
```
to the appropriate place.

Reply:
Changes have been made, facing some failing frontend tests while pushing the code.

Reply:
These are probably flakes, try pushing with `--no-verify` but please create a new issue with the error that you're getting.

Reply:
Sure, thanks! Will open a new issue for the same.

Reply:
Done